### PR TITLE
Added "remember me" to the signup and signin page connecting it to the local storage database for user authentication. 

### PR DESCRIPTION
**Pull request by shriyadindi**
Adding "remember me" to the signup and signin page connecting it to the local storage database for user authentication. 
issue no. : #1675 

https://github.com/user-attachments/assets/bc7eade0-3b9d-47dc-83da-c9170a7e9cfd

### Pull Request Checklist

- [x] I have added screenshots and videos to show before and after the working of my code.
- [x] I have ensured that the screen size is set to 100% while making the video.
- [x] I have synced the latest fork with my local repository and resolved any conflicts.
- [x] I have mentioned the issue number which I created before making this PR .(format to mention issue number is : fixes #(issue number) ) 
- [x] I understand that if any the above conditions are not met , my PR will not be MERGED .

### CODE REVIEW

**Comment by ANSHIKA-26** [comment]
@shriyadindi can you please sync your fork once ? i see a lot of code formatting changes in index.html file which are not even required

**Comment by shriyadindi** [comment]
@ANSHIKA-26 I've synced the fork and now, kindly review it.

File: index.html [hunk 1]
```diff
@@ -460,33 +499,46 @@ <h2 class="text-center">Login</h2>
...
                 <!-- Signup Form -->
                 <div class="form-popup" id="signupForm">
-                    <form class="form-container" onsubmit="return validateSignupForm()" style="max-width: 400px; margin: auto;">
+                    <form class="form-container" onsubmit="return validateSignupForm()"
+                        style="max-width: 400px; margin: auto;">
                         <span class="close-icon" onclick="closeForm('signup')">&times;</span>
                         <h2 class="text-center">Signup</h2>
                         <div class="form-group">
                             <label for="name"><b>Full Name</b></label>
-                            <input type="text" class="form-control" id="name" placeholder="Enter Full Name" name="name" required>
```
Comment (ANSHIKA-26) [inline index.html:468]: same issue here , kindly undo all the unnecessary code formatting you have done

**Review by ANSHIKA-26 — CHANGES_REQUESTED** [review]

**Comment by shriyadindi** [comment]
@ANSHIKA-26 I've removed the formatting done before.

**Pull request merged**